js/views/pauseScreen.js

```diff
@@ -43,17 +48,52 @@ var PauseScreen = createReactClass({
     if (this.state.containsText) {
       this.props.controller.addBlur();
     }
-  },
 
-  componentWillUnmount: function() {
+    this.animateTimer = setTimeout(this.startAnimation, 1);
```

Comment:
Do we need a constant for the timer delay?

Reply:
I just copied this over from the animate mixin. I'll add one in here or in the HOC if we go that route.